index.js

```diff
@@ -114,7 +114,13 @@ module.exports = function create (opts) {
         menubar.window.setPosition(x, y)
       }
 
-      if (!opts['always-on-top']) menubar.window.on('blur', hideWindow)
+      if (!opts['always-on-top']) {
+        menubar.window.on('blur', hideWindow)
+      }
+
+      if (opts['show-on-all-workspaces'] !== false) {
```

Comment:
@maxogden should we go with camelCase or dashes? After we have decided we should make sure all options using the same format. Then I suggest we merge this and #45 and release `v3`.